plugins/source/aws/resources/services/acmpca/certificate_authorities.go

```diff
@@ -0,0 +1,72 @@
+package acmpca
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/acmpca"
+	"github.com/aws/aws-sdk-go-v2/service/acmpca/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/cloudquery/plugin-sdk/v2/transformers"
+)
+
+func CertificateAuthorities() *schema.Table {
+	tableName := "aws_acmpca_certificate_authorities"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/privateca/latest/APIReference/API_CertificateAuthority.html`,
+		Resolver:    fetchAcmpcaCertificateAuthorities,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "acmpca"),
```

Comment:
The name of the service in this location has to match the name that AWS [uses here](https://github.com/cloudquery/cloudquery/blob/66eefda579565b4e76647daf36ca21070d338f59/plugins/source/aws/client/data/partition_service_region.json#L79)

so in this case the service name is `acm-pca`